cryptoconditions/crypto.py

```diff
@@ -143,14 +147,19 @@ def encode(self, encoding='base58'):
         return super().encode(encoder=_get_nacl_encoder(encoding))
 
 
-def ed25519_generate_key_pair():
+def ed25519_generate_key_pair(seed=None):
     """
     Generate a new key pair.
 
+    Args:
+        seed (str) : seed for deterministic generation.
```

Comment:
I tried it and the `seed` must be a 32-byte `bytes` object (not a `str` object). For example:

```ipython
In [10]: myseed = open("/dev/urandom","rb").read(32)                                                                                                                                                        

In [11]: type(myseed)                                                                                                                                                                                       
Out[11]: bytes

In [12]: myseed                                                                                                                                                                                             
Out[12]: b'\x1b\xab\x1d\x98\x1d\xbdSl\xdbC\xf6\x1c\x18\xe7xYE=j=\xa9\xdd_\xcf\x0f\xad\xcf\xb2\x99=s\xca'

In [13]: keypair2 = crypto.ed25519_generate_key_pair(seed=myseed)                                                                                                                                           

In [14]: keypair2                                                                                                                                                                                           
Out[14]: 
(b'2s1Lv1saPMHbTL3F2m6dEfJ99oPtdEqd1bKCHDMvYJpR',
 b'9u98f5YaZ7eB6cVTQXjPgrNWeFF2Eqipb2VFZAo9Zrtq')
```

Reply:
That is, please update this line in the docstring to something like:
```text
        seed (bytes): 32-byte seed for deterministic generation.
```